src_py/apiServer/experiment_flow.py

```diff
@@ -134,14 +134,15 @@ def parse_experiment_flow_json(self, json_path : str):
             self.add_phase(phase_name, phase_type, source_pieces_inst_list)
 
 
-    def generate_experiment_flow_skeleton(self):
-        # Todo implement this function 
+    def generate_experiment_flow_skeleton(self): 
+        # Todo check with david if we need this function
```

Comment:
@ohad123  @NoaShapira8 

The skeleton function generates a json file of experiment flow with fields that user should fill.
It supposed to easier user work. 

We can deal with it later, keep the function with TODO comment.